docs/data/sdks/typescript-browser/index.md

```diff
@@ -757,3 +757,61 @@ If your web app configures the strict Content Security Policy (CSP) for security
+
+For example, if a user logs into the website on one subdomain (`data.amplitude.com`) where the SDK is initialized. When initialization, the SDK assigns cookies to `.amplitude.com`. If the user then navigates to another subdomain (`analytics.amplitude.com`), the login information can be seamlessly shared by shared cookies.
+
+#### Cookie data
+
+The SDK creates two types of cookies: user session cookies and marketing campaign cookies.
+
+???config "User session cookies"
+    |<div class="big-column">Name</div>| Description|
+    |---|----|
+    |`optOut`|Required|
+    |`userId`||
+    |`deviceId`||
+    |`sessionId`||
+    |`lastEventTime`||
+    |`lastEventId`||
+
+???config "Marketing campaign cookies"
+    |<div class="big-column">Name</div>| Description|
+    | --- | --- |
+    |`utm_campaign`| This identifies a specific campaign used (for example, "summer_sale") |
+    |`utm_content` | This identifies what brought the user to the site and is commonly used for A/B testing (for example, "bannerlink", "textlink") |
+    |`utm_id`|An optional parameter for tracking unique IDs or transaction IDs associated with the link.|
+    |`utm_medium`| This identifies a specific campaign used (for example, "summer_sale") |
+    |`utm_source`| This identifies which website sent the traffic (for example, Google, Facebook) |
+    |`utm_term`| This identifies paid search terms used (for example, product+analytics) |
+    |`referrer`|The last page the user was on (for example, `https://amplitude.com/behavioral-analytics-platform?ref=nav`)|
+    |`referring_domain`|The domain that the user was last on (for example, `https://amplitude.com`)|
+    |`dclid`|Google campaign manager Click Identifier|
+    |`gbraid`|Google Click Identifier for iOS device from Web to App|
+    |`gclid`|Google Click Identifier from URL parameters|
+    |`fbclid`|Facebook Click Identifier from URL parameters|
+    |`ko_click_id`|Kochava Click Identifier from URL parameters|
+    |`msclkid`|Microsoft Click Identifier|
+    |`ttclid`|TikTok Click Identifier|
+    |`twclid`|Twitter Click Identifier from URL parameter|
+    |`wbraid`|Google Click Identifier for iOS device from App to Web|
+    |`li_fat_id`|LinkedIn member indirect identifier for Members for conversion tracking, retargeting, analytics|
+    |`rtd_cid`|Reddit Click Identifier| 
+
+#### Disable cookies
+
+By default, `disableCookies` is set to `false` and use `CookieStorage`. You can opt out using cookies by setting `disableCookies` to `true` so that the SDK will use `LocalStorage` instead. `LocalStorage` is a great alternative, but can't track cookies across domains. Because access to `LocalStorage` is restricted by subdomain, you can't track anonymous users across subdomains of your product (for example: `www.amplitude.com` vs `analytics.amplitude.com`).
```

Comment:
```suggestion
You can opt out using cookies by setting `disableCookies` to `true` so that the SDK will use `LocalStorage` instead. `LocalStorage` is a great alternative, but because access to `LocalStorage` is restricted by subdomain, you can't track anonymous users across subdomains of your product (for example: `www.amplitude.com` vs `analytics.amplitude.com`).
```